src/main/java/fr/sncf/osrd/train/events/TrainMove.java

```diff
@@ -0,0 +1,94 @@
+package fr.sncf.osrd.train.events;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import fr.sncf.osrd.simulation.Simulation;
+import fr.sncf.osrd.simulation.SimulationError;
+import fr.sncf.osrd.simulation.TimelineEvent;
+import fr.sncf.osrd.simulation.TimelineEventId;
+import fr.sncf.osrd.train.Interaction;
+import fr.sncf.osrd.train.Train;
+
+public final class TrainMove extends TimelineEvent {
```

Comment:
how about TrainMoveEvent? can you document this class to include the rationale for having two events for the interaction with action points / regular moves?